staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured/helpers.go

```diff
@@ -47,6 +47,9 @@ func NestedFieldNoCopy(obj map[string]interface{}, fields ...string) (interface{
 	var val interface{} = obj
 
 	for i, field := range fields {
+		if val == nil {
+			return nil, false, nil
```

Comment:
Should we give a descriptive error here so that it's clear why the returned object is nil?

Reply:
this is the same case as line 56, where path to the object could not be traversed. we return `false` to indicate the value is not found, not a type error

Reply:
Makes sense. :+1:

/retest